lib/std/os/linux/start_pie.zig

```diff
@@ -16,6 +15,7 @@ const R_RELATIVE = switch (builtin.cpu.arch) {
     .arm => R_ARM_RELATIVE,
     .aarch64 => R_AARCH64_RELATIVE,
     .riscv64 => R_RISCV_RELATIVE,
+    .sparc => R_SPARC_RELATIVE,
```

Comment:
should this be `.sparc` or `.sparcv9`

Reply:
definitely at least `.sparc`. looks like also `.sparcv9`